pages/index.tsx

```diff
@@ -67,7 +77,10 @@ export default function Home() {
           <div className='pointsOfInterest-filter-container' >
             <Button name='info-sidebar-btn' handler={infoSidebarHandler} value='Holder Value'/>          
           </div>
-          {showInfo && <div className='info-sidebar'></div>}
+          {showInfo && <div className='info-sidebar'>
+            {placesInfo.map((place: any) => <Card  id={place.place_id} name={place.name} rating={place.rating} price={place.price_level} address={place.formatted_address} ratings_number={place.user_ratings_total} />)} 
```

Comment:
[pebble] So this file is becoming pretty big. The side bar thing should be broken out into its own component for manageability and reusability

Reply:
Got you fam